content/docs/machine-learning/kubeflow-dashboard.md

```diff
@@ -0,0 +1,57 @@
+---
+sidebar_position: 3
+title: Kubeflow cluster dashboard
+description: Learn the components of Civo's machine learning Kubeflow cluster dashboards
+---
+
+<head>
+  <title>Kubeflow cluster dashboard | Civo Documentation</title>
+</head>
+
+## Logging in to your Kubeflow cluster
+
+Once your cluster is running, you will need to authenticate yourself to access the cluster dashboard. The login credentials to use are the same ones you use to sign in to Civo.
+
+:::noteOther user authentication
+If a Kubeflow cluster administrator has given you access to a cluster they run, you will have a different mode of authentication. Ask your cluster administrator for details on how to log in to the cluster.
+:::
+
+## Kubeflow dashboard home screen
```

Comment:
We are trying to explain the components here. @civo-claire @kaihoffman is it useful to have a diagram in a similar fashion as this (this one is from my blog): https://rishit-dagli.github.io/assets/kubeflow-machine-learning-on-kubernetes-part-1/how-does-kubeflow-fit.png

Reply:
@civo-claire and I were discussing that this is a good starting point for a diagram. We would need to add where Civo is in the diagram (i.e. providing the Kubeflow) but it would be good to get suggestions on how the illustration could work.

Reply:
One of the main things to know first I think would be:

- does Civo Kubeflow deploy all components that Kubeflow traditionally deploys (to make sure we don't add anything extra in the diagram)
- and then have a similar diagram for what parts Civo Kubeflow helps with

What do you think?

Reply:
Are you OK to provide a diagram here @Rishit-dagli ?

Reply:
I could most certainly work on one, do we want something similar to https://rishit-dagli.github.io/assets/kubeflow-machine-learning-on-kubernetes-part-1/how-does-kubeflow-fit.png or just some kind of list of components in Kubeflow? or something else altogether?

Reply:
@Rishit-dagli ideally, both? One for the components and the other for the introduction of what Kubeflow is useful for?